x-pack/plugins/cases/server/client/attachments/validators.ts

```diff
@@ -51,7 +51,13 @@ export const validateFindCommentsPagination = (params?: FindCommentsQueryParams)
   const pageAsNumber = params.page ?? 0;
   const perPageAsNumber = params.perPage ?? 0;
 
-  if (Math.max(pageAsNumber, perPageAsNumber, pageAsNumber * perPageAsNumber) > MAX_DOCS_PER_PAGE) {
+  if (Math.max(perPageAsNumber, 0) > MAX_COMMENTS_PER_PAGE) {
```

Comment:
why the `Math.max`?

Reply:
just in case `params.perPage` is negative

Reply:
hmm if `params.perPage` was negative and we didn't have the `Math.max` it'd still return false right? Do we need the `Math.max`? If so could you add a test case for a negative value.

Reply:
removed `Math.max` as in both case it will return false